src/main/app/fact/FactApi.ts

```diff
@@ -22,6 +22,13 @@ export class FactApi {
       .catch(this.errorHandler({}));
   }
 
+  public updateCourt(slug: string, body: {}): Promise<{}> {
+    return this.axios
+      .put(`/courts/${slug}`, body)
+      .then(results => results.data)
+      .catch(this.errorHandler({}));
```

Comment:
I wonder if we should catch here. Maybe catch in the calling code and display an error to the user?

Reply:
yh, can do that.
there was an issue with displaying success as in the the green banner, i didnt know if we can create our own and the only green banner i see is confirmation panel.